Libraries/Azure.psm1

```diff
@@ -2907,22 +2909,23 @@ Function Upload-AzureBootAndDeploymentDataToDB ($DeploymentTime, $AllVMData, $Cu
 			$null = Copy-RemoteFiles -downloadFrom $vmData.PublicIP -port $vmData.SSHPort -username $user -password $password -files "$($vmData.RoleName)-*.txt" -downloadTo "$LogDir" -download
 
 			# Upload files in data subfolder to Azure.
-			$destfolder = "bootPerf"
-			$containerName = "logs"
-			$blobContext = New-AzStorageContext -StorageAccountName $storageAccountName -StorageAccountKey $storageAccountKey
-
-			$ticks = (Get-Date).Ticks
-			$fileName = "$LogDir\$($vmData.RoleName)-waagent.log.txt"
-			$blobName = "$destfolder/$($fileName.Replace("waagent","waagent-$ticks") | Split-Path -Leaf)"
-			$null = Set-AzStorageBlobContent -File $filename -Container $containerName -Blob $blobName -Context $blobContext -Force
-			$WALAlogFile = "https://$storageAccountName.blob.core.windows.net/$containerName/$destfolder/$($fileName.Replace("waagent","waagent-$ticks") | Split-Path -Leaf)"
-			Write-LogInfo "Upload file to Azure: Success: $WALAlogFile"
-			$fileName = "$LogDir\$($vmData.RoleName)-dmesg.txt"
-			$blobName = "$destfolder/$($fileName.Replace("dmesg","dmesg-$ticks") | Split-Path -Leaf)"
-			$null = Set-AzStorageBlobContent -File $filename -Container $containerName -Blob $blobName -Context $blobContext -Force
-			$kernelLogFile = "https://$storageAccountName.blob.core.windows.net/$containerName/$destfolder/$($fileName.Replace("dmesg","dmesg-$ticks") | Split-Path -Leaf)"
-			Write-LogInfo "Upload file to Azure: Success: $kernelLogFile"
-
+			if ($storageAccountName -and $storageAccountKey) {
```

Comment:
If LISAv2 user does not provide 'bootPerfLogsStorageAccount' or 'bootPerfLogsStorageAccountKey',  just skip this logic and avoid logging error message.